modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -4,18 +4,24 @@
 require 'token_validation/v2/client'
 require 'claims_api/claim_logger'
 require 'claims_api/dependent_service'
+require 'claims_api/v2/params_validation/evidence_waiver_submission'
 
 module ClaimsApi
   module V2
     module Veterans
       class EvidenceWaiverController < ClaimsApi::V2::Veterans::Base
         skip_before_action :validate_json_format
         before_action :set_lighthouse_claim, :set_bgs_claim!, :verify_if_dependent_claim!
+        FORM_NUMBER = '5103'
 
         def submit
+          validate_request!(ClaimsApi::V2::ParamsValidation::EvidenceWaiverSubmission)
           validate_veteran_name(false)
 
+          tracked_item_ids = params['data']['attributes']['trackedItems'] if params['data'].present?
```

Comment:
Should we name the attribute `trackedItemIds` to clarify that we expect an array of integers (not objects)?

Reply:
@FonzMP The [attribute in the DB](https://github.com/department-of-veterans-affairs/vets-api/pull/17563#issuecomment-2256193735) is `tracked_items`, do we want to match that, or change the name to @tycol7's suggestion?

Reply:
I'd agree that we should swap it to `trackedItemids` - not only for the "accepts arrays" but also to align with BD as they take a list of `trackedItemIds`

Reply:
also noted in the ticket
`Add optional trackedItemIds list to GET /5103 request and update documentation`